### PR TITLE
Remove outdated, unused prepare_proxy_test.sh

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

**Comment by jwendell** [comment]
Not sure if removing them is the right thing to do; if it's not, let's just close this PR.

File: pilot/docker/prepare_proxy_test.sh [hunk 1]
```diff
@@ -1,238 +0,0 @@
-#!/bin/bash
```
Comment (ldemailly) [inline pilot/docker/prepare_proxy_test.sh]: I think this was used once upon a time to verify that iptables are working as expected but it clearly rotted

cc @ayj - what do you think ?

**Comment by ldemailly** [comment]
this could be revived to test #3562

File: pilot/docker/prepare_proxy_test.sh (hunk 1 above)
Comment (ayj) [reply pilot/docker/prepare_proxy_test.sh]: https://github.com/istio/istio/pull/3562 probably ought to include an equivalent unit test.

Comment (ayj) [reply pilot/docker/prepare_proxy_test.sh]: Also, `prepare_proxy.sh` hasn't experienced much churn. I suspect this test isn't that far out of date.

**Comment by jwendell** [comment]
fair enough, sorry for the noise :)

File: pilot/docker/prepare_proxy_test.sh (hunk 1 above)
Comment (ldemailly) [reply pilot/docker/prepare_proxy_test.sh]: agreed, just that I noticed stuff like `bazel run...`

**Comment by ldemailly** [comment]
@jwendell nah it's good - the issue is the test isn't exercised, otherwise I totally agree we should remove dead code and duplicate, continue to fight the good fight!

**Pull request closed without merging**